camelot/view/action_steps/update_progress.py

```diff
@@ -111,7 +111,7 @@ def gui_run(cls, gui_context_name, serialized_step):
                 # reset progress dialog
                 reset_step = QtCore.QByteArray(json.dumps({ 'reset': True }).encode())
                 qml_action_step(gui_context_name, 'UpdateProgress', reset_step)
-                post(action_runner._cancel_request)
+                post(CancelAction(run_name=[]))
```

Comment:
Hebben we hier eigenlijk toegang tot de run_name? Of moeten we hier toch een CancelRequest exception raisen, die opvangen in AbstractRequest._iterate_until_blocking en dan die CancelAction request posten met de run_name?

Reply:
in principe kan je daar toegang hebben tot de run_name

ik heb dat voor deze aanpassing gelaten 'as is', omdat er met de run_name toch niks gedaan w (brute force cancel)

maar naar de toekomst toe, zou het beter zijn om die run_name daar mee te geven, en dan ook geen brute force cancel meer te doen.

een exception raisen in gui code zou ik niet doen, gezien we in C++ zonder exceptions werken ... anders w de code weer lastiger om te porten ...